cmd/up_test.go

```diff
@@ -209,7 +209,14 @@ func TestDump(t *testing.T) {
 	objs = append(objs, obj)
 	var buf bytes.Buffer
 
-	err := dump(&buf, "yaml", objs)
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
```

Comment:
If you're happy using empty schemas for this test, then I think you can also use the standard fake Discovery client here:
```go
client := fake.NewSimpleClientset()

err := dump(&buf, "yaml", client.Discovery(), objs)
...
￼```

This fake class should be slightly faster/simpler, since it doesn't actually run an HTTP server or serialise/deserialise objects.

Reply:
> If you're happy using empty schemas for this test

Perhaps this test should do more, but I guess since sort is tested in kubecfg we can rely on that and just test that we get some sort of desired output. I would say we need to extend the desired output test and ensure we get the correct format for JSON and YAML, but that's a separate PR.

Thanks for the tip, will use the fake disco client!